frontend/src/components/subject/subjects/header/SubjectsHeaderCard.vue

```diff
@@ -15,7 +15,7 @@
                             color="secondary"
                             variant="flat"
                         >
-                            {{ `${academicYear}-${academicYear + 1}` }}
+                            {{ `${academicYear - 2000}-${academicYear - 2000 + 1}` }}
```

Comment:
Ik vindt het niet zo intuïtief dat de knop van het geselecteerde jaar donder is, en de andere knop wit.